lib/daisy_ui_components/tabs.ex

```diff
@@ -0,0 +1,250 @@
+defmodule DaisyUIComponents.Tabs do
+  @moduledoc """
+  Tab component
+
+  Renders a list of tab links or toggles with radio toggle and content or label with radio toggle and content
+
+  https://daisyui.com/components/tab/
+  """
+
+  use DaisyUIComponents, :component
+
+  @doc """
+  Renders a Tabs with links as toggle by default.
+
+  ## Default Example
+  <.tabs>
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab disabled>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div role="tablist" class="tabs">
+    <a role="tab" class="tab">Tab 1</a>
+    <a role="tab" class="tab tab-active">Tab 2</a>
+    <a role="tab" class="tab tab-disabled">Tab 3</a>
+  </div>
+
+  ## Radio Example
+  ## https://daisyui.com/components/tab/#radio-tabs-border--tab-content
+  <.tabs type="radio">
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div class="tabs">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 1">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 2" checked="checked">
+    <input type="radio" name="tabs" class="tab" aria-label="Tab 3">
+  </div>
+
+  ## Label Example
+  ## https://daisyui.com/components/tab/#radio-tabs-lift-with-icons--tab-content
+  <.tabs type="label">
+    <:tab>Tab 1</:tab>
+    <:tab active>Tab 2</:tab>
+    <:tab>Tab 3</:tab>
+  </.tabs>
+
+  ## Renders
+  <div class="tabs">
+    <label class="tab">
+      <input type="radio" name="tabs" class="tab"> Tab 1
+    </label>
+    <label class="tab">
+      <input type="radio" name="tabs" class="tab" checked="checked"> Tab 2
+    </label>
+    <label class="tab">
+      <input type="radio" name="tabs" class="tab"> Tab 3
+    </label>
+  </div>
+  """
+
+  attr :type, :string, values: ~w(link radio label), default: "link"
+  attr :name, :string, default: nil
+  attr :class, :any, default: nil
+  attr :size, :string, values: sizes()
+  attr :tabs_style, :string, values: ~w(box border lift)
+  attr :tabs_position, :string, values: ~w(top bottom)
+  attr :rest, :global
+
+  slot :tab do
+    attr :active, :boolean
+    attr :disabled, :boolean
+  end
+
+  slot :tab_content
+
+  slot :inner_block
+
+  def tabs(%{type: "radio"} = assigns) do
+    assigns =
+      assign(assigns, :class, tabs_classes(assigns))
+
+    ~H"""
+    <div class={@class} {@rest}>
+      <%= for {tab, index} <- Enum.with_index(@tab) do %>
```

Comment:
I think you don't need the index from `Enum.with_index(@tab)`

```heex
<%= for tab <- @tab do %>
        <.tab_radio
          name={@name}
          title={@title} # adding the new title attribute for naming the tabs
          class={radio_tab_classes(tab)}
          aria_label={render_slot(tab)}
          active={tab[:active]}
        />
        <.tab_content :if={tab_content = render_slot(@tab)} class={tab_content_classes(tab_content)}>
          {tab_content}
        </.tab_content>
<% end %>
```

This would simplify the logic also for the content, no need for the slot `slot :tab_content` in the component attributes.